server/src/main/java/org/elasticsearch/action/admin/indices/close/TransportVerifyShardBeforeCloseAction.java

```diff
@@ -151,21 +162,32 @@ public void markShardCopyAsStaleIfNeeded(ShardId shardId, String allocationId, A
     public static class ShardRequest extends ReplicationRequest<ShardRequest> {
 
         private final ClusterBlock clusterBlock;
+        private final boolean phase1;
 
         ShardRequest(StreamInput in) throws IOException {
             super(in);
+            if (in.getVersion().onOrAfter(Version.V_4_3_0)) {
```

Comment:
The transport and request is new in 4.3 and wasn't present in 4.2 so the check here can be removed.

There is logic in place to avoid using the transport if not all nodes are on 4.3